frontend/src/components/common/DownloadMenu.tsx

```diff
@@ -0,0 +1,54 @@
+import { SaveAlt as SaveAltIcon } from "@mui/icons-material";
+import { IconButton, Menu, MenuItem, Paper } from "@mui/material";
+import { MouseEvent, useCallback, useState } from "react";
+import { useFileExport } from "../../hooks/useFileExport";
+
+const DownloadMenu = () => {
+	const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
+
+	const handleClick = (event: MouseEvent<HTMLButtonElement>) => {
+		setAnchorEl(event.currentTarget);
+	};
+
+	const handleClose = () => {
+		setAnchorEl(null);
+	};
+
+	const { exportToPDF, exportToTXT, exportToDOCX } = useFileExport();
+
+	const handleExportToPDF = useCallback(() => {
```

Comment:
It seems necessary to standardize either wrapping or not wrapping handlers in the Component with `useCallback`.